clients/java/src/main/java/io/camunda/client/impl/search/request/SearchRequestPageImpl.java

```diff
@@ -43,13 +44,14 @@ public SearchRequestPage limit(final Integer value) {
 
   @Override
   public SearchRequestPage searchBefore(final List<Object> values) {
-    page.setSearchBefore(values);
+    page.setSearchBefore(PageObjectMapper.fromObjectList(values));
     return this;
   }
 
   @Override
   public SearchRequestPage searchAfter(final List<Object> values) {
-    page.setSearchAfter(values);
+
+    page.setSearchAfter(PageObjectMapper.fromObjectList(values));
     return this;
   }
```

Comment:
❓ Shouldn't we fix this to a `List<PageObject>`? That would avoid some ambiguities and potential misuse, I believe. Users should only put `PageObject`s in the `searchBefore`/`searchAfter`, shouldn't they? Why would I need a method that accepts a list of raw values in the client? Where would those come from? I am only supposed to put in the list I received from a REST server response. The server response should contain a list of `PageObject`s.